Assets/Scripts/Audio/SFX/SFXController.cs

```diff
@@ -65,7 +64,7 @@ public void PlayRocketLaunchCooldownSFX (AudioSource source)
         source.Stop();
         source.clip = missileSounds[0];
         source.Play();
-        StartCoroutine(PlayAfterDelay(source, combatSounds[1], 1f, .3f));
+        StartCoroutine(PlayAfterDelay(source, combatSounds[1], 1f, .1f));
```

Comment:
I know this is old code but I didn't notice before. I'm not a fan of always using a predefined index here, if combatSounds has a bunch of sounds that should be used at different times shouldn't they all be named for those different times?

Reply:
lol fuckin ludum jank, idk how I saw that system yesterday and just chose to ignore it

Reply:
https://github.com/jacobmcleman/LD46/pull/30/commits/fb0ede07ee286d0a37be569db8cbbebc6fc351c9